neuralmonkey/decoders/beam_search_decoder.py

```diff
@@ -231,7 +253,7 @@ def body(*args) -> BeamSearchLoopState:
             step = dec_loop_state.feedables.step - 1
 
             # mask the probabilities
-            # shape(logprobs) = beam x vocabulary
+            # shape(logprobs) = (batch * beam) x vocabulary
```

Comment:
tady to máš s mezerou a nahoře tohohle souboru bez mezery, tak to pls sjednoť

Reply:
Ok, to nasobeni '*' bude bez mezery, at je to trochu vizualne odlisene.